### ISSUE TITLE
[ENH] consolidate the many places where dependencies are listed

### ISSUE THREAD

**Issue by ericzolf** [rdiff-backup/rdiff-backup#730]
## Is your feature request related to a problem? Please describe.

As I need to modify dependencies, I noticed in how many different places they are defined.

## Describe the solution you'd like

Find a way to centralize the dependencies

## Describe alternatives you've considered

I've already started to document where to look for dependencies, but it's still not perfect.

## Additional context

I'm a lazy bastard :smile_cat:

**Comment by ericzolf** [comment]
Need also to consider which dependencies are fixed, which aren't.

**Comment by ericzolf** [comment]
This could prove useful: https://stackoverflow.com/questions/12258456/how-can-tox-install-the-modules-via-the-requirements-file
tl;dr:

```
[testenv]
 deps = -Ur{toxinidir}/requirements.txt
        -Ur{toxinidir}/requirements.testing.txt
```
